Dockerfile

```diff
@@ -1,13 +1,14 @@
 FROM fedora:28
 
 MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
+ARG LIBVIRT_VERSION=4.9.0
 ENV container docker
 
 RUN curl --output /etc/yum.repos.d/fedora-virt-preview.repo \
-  https://fedorapeople.org/groups/virt/virt-preview/fedora-virt-preview.repo && \
+  https://copr.fedorainfracloud.org/coprs/g/virtmaint-sig/virt-preview/repo/fedora-28/group_virtmaint-sig-virt-preview-fedora-28.repo && \
```

Comment:
You should instead do `dnf copr enable @virtmaint-sig/virt-preview` as seen here:
https://fedoraproject.org/wiki/Virtualization_Preview_Repository